colorbox.admin.inc

```diff
@@ -147,101 +125,105 @@ function colorbox_admin_settings() {
     '#type' => 'checkbox',
-    '#title' => t('Fixed'),
+    '#title' => t('Fixed position'),
     '#default_value' => config_get('colorbox.settings', 'colorbox_fixed'),
     '#description' => t('If the Colorbox should be displayed in a fixed position within the visitor\'s viewport or relative to the document.'),
   );
-
-  $form['colorbox_custom_settings']['colorbox_slideshow_settings'] = array(
-    '#type' => 'fieldset',
-    '#title' => t('Slideshow settings'),
-    '#collapsible' => TRUE,
-    '#collapsed' => TRUE,
-  );
-  $form['colorbox_custom_settings']['colorbox_slideshow_settings']['colorbox_slideshow'] = array(
-    '#type' => 'radios',
-    '#title' => t('Slideshow'),
-    '#options' => array(0 => t('Off'), 1 => t('On')),
-    '#default_value' => config_get('colorbox.settings', 'colorbox_slideshow'),
-    '#description' => t('An automatic slideshow to a content group / gallery.'),
-    '#prefix' => '<div class="colorbox-slideshow-settings-activate">',
-    '#suffix' => '</div>',
-  );
   $form['colorbox_custom_settings']['colorbox_scrolling'] = array(
     '#type' => 'radios',
     '#title' => t('Scrollbars'),
     '#options' => array(1 => t('On'), 0 => t('Off')),
     '#default_value' => config_get('colorbox.settings', 'colorbox_scrolling'),
-    '#description' => t('If false, Colorbox will hide scrollbars for overflowing content. This could be used on conjunction with the resize method for a smoother transition if you are appending content to an already open instance of Colorbox.'),
+    '#description' => t('If set to Off, Colorbox will hide scrollbars for overflowing content. This could be used in conjunction with the resize method for a smoother transition if you are appending content to an already open instance of Colorbox.'),
   );
 
-  $js_hide = config_get('colorbox.settings', 'colorbox_slideshow') ? '' : ' js-hide';
-  $form['colorbox_custom_settings']['colorbox_slideshow_settings']['wrapper_start'] = array(
-    '#markup' => '<div class="colorbox-slideshow-settings' . $js_hide . '">',
+  $form['colorbox_slideshow'] = array(
+    '#type' => 'radios',
+    '#title' => t('Automatic Slideshow'),
+    '#options' => array(
+      0 => t('Off'),
+      1 => t('On'),
+    ),
+    '#default_value' => config_get('colorbox.settings', 'colorbox_slideshow'),
+    '#description' => t('An automatic slideshow to a content group / gallery.'),
   );
-
-  $form['colorbox_custom_settings']['colorbox_slideshow_settings']['colorbox_slideshowauto'] = array(
+  $form['colorbox_slideshow_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Slideshow settings'),
+    '#states' => array(
+      'visible' => array(
+        ':input[name="colorbox_slideshow"]' => array('value' => '1'),
+      ),
+    ),
+  );
+  $form['colorbox_slideshow_settings']['colorbox_slideshowauto'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Slideshow autostart'),
+    '#title' => t('Automatically start to play the slideshow.'),
```

Comment:
Description/help text must have periods. Checkbox text and labels should not AFAIK.